File: common/protcol/js/module/hub_call_clientmodule.js
/*this module file is codegen by juggle for js*/
function hub_call_client_module(){
    eventobj.call(this);
    Imodule.call(this, "hub_call_client");

    this.call_client = function(argv0, argv1, argv2){
        this.call_event("call_client", [argv0, argv1, argv2]);
    }

}
(function(){
    var Super = function(){};
    Super.prototype = Imodule.prototype;
    hub_call_client_module.prototype = new Super();
})();
hub_call_client_module.prototype.constructor = hub_call_client_module;

File: protcol/c#/module/hub_call_clientmodule.cs
/*this module file is codegen by juggle for c#*/
using System;
using System.Collections;
using System.Collections.Generic;

namespace module
{
    public class hub_call_client : juggle.Imodule 
    {
        public hub_call_client()
        {
			module_name = "hub_call_client";
        }

        public delegate void call_clienthandle(String argv0, String argv1, ArrayList argv2);
        public event call_clienthandle oncall_client;
        public void call_client(ArrayList _event)
        {
            if(oncall_client != null)
            {
                var argv0 = ((String)_event[0]);
                var argv1 = ((String)_event[1]);
                var argv2 = ((ArrayList)_event[2]);
                oncall_client( argv0,  argv1,  argv2);
            }
        }

	}
}
